docs/agent-integrations.mdx

```diff
@@ -494,6 +494,46 @@ Add to `~/.gemini/settings.json` or `.gemini/settings.json`:
   Configuration](https://github.com/google-gemini/gemini-cli/blob/main/docs/cli/configuration.md)
 </Info>
 
+## JetBrains Junie
+
+### Add MCP Configuration
+
+In the settings, under Tools → Junie → MCP Settings add or edit the MCP configuration, or by editing the file `~/.junie/mcp/mcp.json`:
```

Comment:
```suggestion
Add or edit the MCP configuration under Tools → Junie → MCP Settings, or edit `~/.junie/mcp/mcp.json`:
```

Reply:
What about something like the following?

```markdown
For JetBrains IDEs (see the full compatible list [here](https://www.jetbrains.com/junie/), add or edit the MCP configuration in Settings under Tools → Junie → MCP Settings, or edit `~/.junie/mcp/mcp.json`:
```

It should work with all of their IDEs with Junie and not only IntelliJ. I've tested it with Goland for instance.